README.md

```diff
@@ -17,22 +17,25 @@ the [CC0 1.0 Universal public domain dedication](https://creativecommons.org/pub
 
 For more information, see [license](LICENSE.md).
 
-
 ## Deployment
 
-Refer to [docs.18f.gov](https://docs.18f.gov/getting-started/setup/) for getting set up with Cloud Foundry
-
+Refer to [docs.18f.gov](https://docs.18f.gov/getting-started/setup/) for getting
+set up with Cloud Foundry
 
 To deploy:
-`cf target -o 18f -s dolores` 
 
-`cf set-env dolores HOST dolores-app.18f.gov`
-`cf set-env dolores APPLICATION_HOST dolores-app.18f.gov`
-`cf set-env dolores ASSET_HOST dolores-app.18f.gov`
-`cf set-env dolores DEFAULT_URL_HOST dolores-app.18f.gov`
+```bash
+cf target -o 18f -s dolores
+cf set-env dolores HOST dolores-app.18f.gov
+cf set-env dolores APPLICATION_HOST dolores-app.18f.gov
+cf set-env dolores ASSET_HOST dolores-app.18f.gov
+cf set-env dolores DEFAULT_URL_HOST dolores-app.18f.gov
+```
 
 To push to production:
+
 `cf push dolores`
```

Comment:
I fixed this on my deployment branch, but since you are doing it here. Line 37 should be `cf push dolores-app`